plugin/storage/es/esCleaner.py

```diff
@@ -59,6 +59,7 @@ def main():
 
 def filter_main_indices(ilo, prefix):
     ilo.filter_by_regex(kind='prefix', value=prefix + "jaeger")
+    empty_list(ilo, "No indices to delete")
```

Comment:
`empty_list` function seems to kill the process if the list is empty (better name would be `panic_if_list_empty`)

so how does this achieve "do not fail" objective?

Reply:
`empty_list` ends the process with `0` which indicates success. This is exactly what we want. Without this fix the process is not ended and an exception is thrown.